oshi-core/src/main/java/oshi/util/FileUtil.java

```diff
@@ -151,17 +151,6 @@ public static List<String> readLines(String filename, int count, boolean reportE
         return Collections.emptyList();
     }
 
-    /**
-     * Read an entire file at one time. Intended primarily for Linux /proc filesystem to avoid recalculating file
-     * contents on iterative reads.
-     *
-     * @param filename The file to read
-     * @return A byte array representing the file
-     */
-    public static byte[] readAllBytes(String filename) {
```

Comment:
💭 This method was now unused, and I think it is better to remove it since you should always think whether or not you want to log errors.

Reply:
Yeah, this method was a relic from pre-Java8 days.  The JDK has an equivalent if we ever need it again.